deployments/virtual-services.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ui-service
+spec:
+  hosts: ui-app
+  gateways: api-gateway
+  http:
+  - route:
+    - destination:
+        host: ui-app
+        subset: v1
+      weight: 80
+    - destination:
+        host: ui-app
+        subset: v2
+      weight: 20		
+---
+
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: session-management
+spec:
+  hosts:
+  - session-management
+  gateways:
+  - captivate-gateway
+  http:
+  - route:
+    - destination:
+        host: session-management
+
+---
+
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: user-management
+spec:
+  hosts:
+  - user-management
+  gateways:
+  - captivate-gateway
+  http:
+  - route:
+    - destination:
+        host: user-management
```

Comment:
May I know why we need these ?

Reply:
can be removed if we don't want to define traffic rules for these services